Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -59,7 +58,7 @@ def pr_automerge
     end
 
     if args.publish?
-      safe_system "#{HOMEBREW_PREFIX}/bin/brew", "pr-publish", *pr_urls
```

Comment:
We'd need to pass along a `--tap` argument when relevant here.

Reply:
Done